### PR TITLE
add option -e for measuring the audio engine performance without in a

### PR DESCRIPTION
**Pull request by hhoegelo**
reproducible way

### CODE REVIEW

**Comment by MatthiasSeeber** [comment]
I've tried several command line variations (with "-e", "-e 0", -"e 1" or without) and everytime I get this message and the program terminates:
Error:  Alsa Error: Invalid argument 
Info:  Alsa Frames per Period: 480 
Info:  Alsa Periods: 2 
Info:  Midi2Audio latency is: 10 ms. 
Press <RETURN> to close this window...
Am I doing something wrong?

File: audio-engine/CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +51,7 @@ endforeach(lib)
 
 include_directories(src)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17 -Wdouble-promotion")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17 -Wdouble-promotion -march=broadwell -Ofast")
```
Comment (AntonSchmied) [inline audio-engine/CMakeLists.txt]: if i leave out the flags after the word "promotion", only then i can measure the performance ... 
Is it possible that it is a bit to specific for a general test?

**Comment by AntonSchmied** [comment]
`set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17 -Wdouble-promotion -march=broadwell -Ofast")`

If I leave out the the flags "-march=broadwell" and "-Ofast", only then I can test the performance.
Is this maybe too specific and not suitable for general tests?

**Comment by MatthiasSeeber** [comment]
...what Anton is saying, now it's working.

**Pull request merged**